README.md

```diff
@@ -254,7 +254,8 @@ This module is your main entry-point when asserting.
    then fail with `format` as an error message together with a description
    of `subject` and `length`.
  - `contains : 'a seq -> 'a -> string -> unit` – Expect the sequence to contain the item.
- - `containsAll: 'a seq -> 'a seq -> string -> unit` - Expect the sequence contains all elements from second sequence (not taking into account an order of elements)
+ - `containsAll: 'a seq -> 'a seq -> string -> unit` - Expect the sequence contains all elements from second sequence (not taking into account an order of elements and number of occurances of elements)
+ - `distributed: 'a seq -> Map<'a, int> -> string -> unit` - Expect the sequence contains all elements from map (first element in tuple is an item expected to be in sequence, second one is a number of it's occurances in sequence). Sequence is not taking into account an order of elements.
```

Comment:
Also needs to be updated. "Expect [that] the ..." Avoid mixing numbers with colloquial speech: "second one". "its", not "it's". Also, is it the "sequence" that takes into account – not the Expect-module function? Also, if something *is* taking something into account, should it not be the Map that specifies the distribution?

Should we consider just taking a `(string * string) seq` instead of a `Map` to avoid the Map c'tor call at the call-site?